components/SelectorBase/SelectorBase.behavior.comp.cy.js

```diff
@@ -123,18 +123,76 @@ describe('Test the SelectorBase behaviors', () => {
         invalidFeedbackText: 'Invalid feedback text.',
         label: `TheLabel`,
         options: ['One', 'Two', 'Three', 'Four', 'Five'],
-        includeAddButton: true,
         onAddClicked: addClickedSpy,
         onReady: readySpy,
+        popupUrl: '',
       },
     });
 
     cy.get('@readySpy')
       .should('have.been.calledOnce')
       .then(() => {
+        cy.get('[data-cy="selector-overlay"]').should('not.exist');
+        cy.get('[data-cy="selector-popup"]').should('not.exist');
+        cy.get('[data-cy="selector-closePopup"]').should('not.exist');
+        cy.get('[data-cy="selector-popupIframe"]').should('not.exist');
+
         cy.get('[data-cy="selector-add-button"]').should('exist');
         cy.get('[data-cy="selector-add-button"]').click();
+
+        cy.get('[data-cy="selector-overlay"]').should('exist');
+        cy.get('[data-cy="selector-popup"]').should('exist');
+        cy.get('[data-cy="selector-closePopup"]').should('exist');
+        cy.get('[data-cy="selector-popupIframe"]')
+          .should('exist')
+          .should('have.attr', 'src', '');
+
+        cy.get('[data-cy="selector-closePopup"]').click();
         cy.get('@addClickedSpy').should('have.been.calledOnce');
+        cy.get('[data-cy="selector-overlay"]').should('not.exist');
+        cy.get('[data-cy="selector-popup"]').should('not.exist');
+        cy.get('[data-cy="selector-closePopup"]').should('not.exist');
+        cy.get('[data-cy="selector-popupIframe"]').should('not.exist');
+      });
+  });
+  it('Clicking add button disables background elements and close enables them', () => {
+    const readySpy = cy.spy().as('readySpy');
+
+    cy.mount(SelectorBase, {
+      props: {
+        required: true,
+        invalidFeedbackText: 'Invalid feedback text.',
+        label: `TheLabel`,
+        options: ['One', 'Two', 'Three', 'Four', 'Five'],
+        onReady: readySpy,
+        popupUrl: '',
```

Comment:
Can this also be a `nonNullValue`?  More generally can any uses of `''` as the URL be changed to reduce changes of confusion?  If there is a reason they cannot just add an explanation.

Reply:
It seems to cause an error during testing when ever the (+) is clicked and the iframe has its source set. I commented out all the code i added which tampers with the iframe (checking URL, removing elements, etc.) and even with a plain iframe it caused errors. Since these tests do not test for the content of the iframe (that is tested the e2e tests) i think empty string is probably sensible solution.